src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +22,15 @@ public class StreamPractice {
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        if (numbers.isEmpty()) {
+            throw new RuntimeException(EMPTY_LIST);
+        }
+        return numbers.stream()
+                .flatMap(n -> Arrays.stream(n.split(NUMBERS_SEPARATOR)))
+                .filter(n -> Integer.parseInt(n) % 2 == 0)
```

Comment:
I think, we can parseInt first and filter, then we won't need to parse twice 
```suggestion
                .mapToInt(Integer::parseInt)
                .filter(n -> n % 2 == 0)
```